frontend/src/views/ShootList.vue

```diff
@@ -16,6 +16,41 @@ SPDX-License-Identifier: Apache-2.0
           <div class="text-subtitle-1 toolbar-title--text">{{headlineSubtitle}}</div>
         </v-toolbar-title>
         <v-spacer></v-spacer>
+        <v-tooltip top>
```

Comment:
```suggestion
        <v-tooltip bottom>
```

otherwise the tooltip overlaps the badge